StoreApp/Views/Shared/_Layout.cshtml

```diff
@@ -43,7 +43,10 @@
             <ul class="navbar-nav">
                 @if (Request.IsAuthenticated)
                 {
-                    <li class="nav-item">@Html.Action("CartPartial", "Cart")</li>
+                    if (!Session["Role"].ToString().Equals("admin"))
```

Comment:
You can call Enum here. 
Check this statement below please

Reply:
fixed